modules/serverless-configuring-burst-qps-for-net-kourier.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/kube-burst-qps-net-kourier.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-burst-qps-for-net-kourier_{context}"]
+= Configuring burst and QPS values for net-kourier
+
+The queries per second (QPS) value determines the number of client requests or API calls that are sent to the API server.
+
+The burst value determines how many requests the client can make in excess of the QPS rate. This is helpful for controllers that are bursty in nature and do not spread their requests uniformly in time.
```

Comment:
As I read this first sentence it's not fully clear to me how "burst" works. I think we could make it more clear in this way: `The burst value determines how many requests from the client can be stored for processing. Requests exceeding this buffer will be dropped.`